.minio.sys/buckets/browser/policy.json

```diff
@@ -0,0 +1 @@
+{"Version":"2012-10-17","Statement":[{"Action":["s3:GetBucketLocation"],"Effect":"Allow","Principal":{"AWS":["*"]},"Resource":["arn:aws:s3:::browser"],"Sid":""},{"Action":["s3:ListBucket"],"Condition":{"StringEquals":{"s3:prefix":["/docs","cmd","fefe fef "]}},"Effect":"Allow","Principal":{"AWS":["*"]},"Resource":["arn:aws:s3:::browser"],"Sid":""},{"Action":["s3:GetObject"],"Effect":"Allow","Principal":{"AWS":["*"]},"Resource":["arn:aws:s3:::browser//docs*","arn:aws:s3:::browser/cmd*","arn:aws:s3:::browser/fefe fef *"],"Sid":""}]}
```

Comment:
You shouldn't add these files into your commit.  Also make sure to run minio on a separate directory.